### DISCUSSION TITLE
更改容器名称后，发现原来的账户失效了，虽然数据库还在，但是无法再登录之前的账户。

### DISCUSSION THREAD

**Discussion by leoneoo** [usememos/memos#227] [General]
更改容器名称后，发现原来的账户失效了，虽然数据库还在，但是无法再登录之前的账户。有何办法能恢复账户？或者旧的数据库能否绑定新的账户？

**Comment by boojack** [comment]
可以发一下你最新的启动命令吗

**Reply by leoneoo** [reply]
我在群晖上重新测试了一遍，应该是升级容器造成的。由于docker本身无法升级容器，我参照了以下方法，升级了容器，https://www.bilibili.com/video/BV15Z4y1i7g4?vd_source=cdd7ec186abdcdde8d8f589006e13607     导致数据库无法关联账户。这样操作的话，数据库虽然还在，设置也没有更改，但是实际上等于重新初始化了账户。还请指导如何正确升级memos版本，又不会导致数据丢失。

**Reply by leoneoo** [reply]
最好能像flomo一样，能够导出数据，解决数据备份的问题。

**Reply by boojack** [reply]
> 这样操作的话，数据库虽然还在，设置也没有更改，但是实际上等于重新初始化了账户。

你是说版本更新后用户表数据被篡改了？

> 最好能像flomo一样，能够导出数据，解决数据备份的问题。

数据库只有一个文件，可以直接对这个文件进行备份，不需要额外再导出导入。

**Reply by leoneoo** [reply]
差不多这样！原先本来注册的账户已经无法再次登录了，直接提示“sign up as host”，这样就需要重新注册新账户啦！

**Reply by boojack** [reply]
感觉可能是没把数据文件路径挂载进去，可以先用某个数据库管理工具看看你的数据是否正常。
